### PR TITLE
German translations for "Rising Sun"

### PR DESCRIPTION
**Pull request by s3inlc**
Hey there :)

I added the German translations for all the cards of the "Rising Sun" expansion, which hopefully others are interested to get as well.

Please let me know if there is anything else I need to add or change in order to make the PR correctly.

### CODE REVIEW

**Comment by sumpfork** [comment]
> Hey there :)
> 
> I added the German translations for all the cards of the "Rising Sun" expansion, which hopefully others are interested to get as well.
> 
> Please let me know if there is anything else I need to add or change in order to make the PR correctly.

Vielen Dank!

Could you re-compile the card db one more time, I think it just needs your small fix (second commit in this PR) integrated perhaps?

**Comment by s3inlc** [comment]
Ah indeed, I may have forgot that after the small change. I re-ran the update_language command again and pushed.

File: card_db_src/de/sets_de.json [hunk 1]
```diff
@@ -240,6 +240,11 @@
         "set_text": "",
         "text_icon": "R"
     },
+    "risingsun": {
+        "set_name": "Rising Sun",
+        "set_text": "Wir segeln jetzt zu den Inseln im Osten - oder Westen, das hängt davon ab, wo du dich im Verhältnis zu ihnen befindest. Dein Titel ist hier \"Kaiser\". Man sagte dir, du bist zwar nur ein Grüßaugust, aber du kannst trotzdem zum Frühstück alles bestellen, was du willst. Vielleicht haben sie recht, du hast dein gewünschtes Frühstück bekommen. Dein Zeremonienschwer ist nur aus Pappe, genau wie die Rüstung. Die Samurai lassen dich nie zu ihren Teepartys und die Ninjas binden immer deine Schnürsenkel zusammen. Und das epische Gedicht, das sie dir widmeten, hat nur 17 Silben. Reis wird jetzt als Währung benutzt, und niemand versucht auch nur, dein Gesicht auf die Körner zu bekommen. Aber wenn du morgens aufwachst und über dein Land schaust, findest du dein Leben gar nicht so schlecht. Also, was gibt's heute zum Frühstück?",
```
Comment (sumpfork) [inline card_db_src/de/sets_de.json:245]: Zeremonienschwer -> Zeremonienschwert

**Review by sumpfork — COMMENTED** [review]
Ich habe nicht alles durchgelesen, aber zufällig doch ein fehlendes "t" bemerkt.

File: card_db_src/de/sets_de.json (hunk 1 above)
Comment (s3inlc) [reply card_db_src/de/sets_de.json:245]: Ah, in der Tat, da fehlt was. Soll ich das ändern und nochmals commiten, oder warten, ob noch ein weiterer Typo auftaucht?

Comment (sumpfork) [reply card_db_src/de/sets_de.json:245]: Ich denke einfach kurz verbessern und dann werde ich ein Deployment machen, falls andere später Typos finden können die dann korrigiert werden.

**Review by sumpfork — APPROVED** [review]

**Pull request merged**